Gemfile

```diff
@@ -4,7 +4,7 @@ gem 'resque', '~>1.19'
 
 group :test do
   gem 'mocha', '~>0.9'
-  gem 'shoulda', '~>2.10.2'
+  gem 'minitest'
```

Comment:
changed, same argument could be made for jeweler since version 0.1 had a different api etc ... so I think it's not necessary